components/fleetArchiveDS.js

```diff
@@ -141,7 +141,6 @@ class FleetArchiveLogs extends Datasource {
       .map(
         (entry) =>
           _.get(entry, "createTrip.response.vehicleId") ||
-          _.get(entry, "getTrip.response.vehicleId") ||
```

Comment:
I don't think Cloud Logging filters out getTrip calls? Can you tell me more?

Reply:
Nevermind, I see a filter here https://github.com/googlemaps/fleet-debugger/blob/5f67d794f24aa276ad97ee72368a987861f65731/components/logging.js#L79

Reply:
Yes, also get_trip isn't in the "interestingLogs" list so the fetching code already might not be getting them anymore.